### PR TITLE
add router and bottoms

### PR DESCRIPTION
**Pull request by Yesika-Castillo**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,48 @@
...
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="body">
+        <div className="Buttons">
+          <button
+            className={`botton ${isActive === 'list' && 'bold-button'}`}
```
Comment (Lizhtml) [inline src/App.js:13]: Please check the condition, "false" is added as a CSS class when the condition doesnt' pass.
Maybe try something like:
  className={`botton ${isActive === 'list' ? 'bold-button':''}`}

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,48 @@
...
+            onClick={() => setIsActive('list')}
+          >
+            <Link to="/list" className="no-decoration">
+              List
+            </Link>
+          </button>
+          <button
+            className={`botton ${isActive === 'add' && 'bold-button'}`}
```
Comment (Lizhtml) [inline src/App.js:21]: Same here.

File: src/App.css [hunk 3]
```diff
@@ -1,22 +1,40 @@
...
-  text-align: center;
+.Buttons {
+  display: flex;
+  justify-content: space-around;
+  background-color: #282c34;
 }
 
-.App-logo {
-  height: 40vmin;
+.botton {
```
Comment (alejandronanez) [inline src/App.css:7]: This should be `button` instead 😅

File: src/App.js [hunk 4]
```diff
@@ -1,26 +1,48 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
 import './App.css';
 
-function App() {
+export default function App() {
+  const [isActive, setIsActive] = useState('list');
```
Comment (alejandronanez) [inline src/App.js:6]: Nice, good work with hooks here! :)

File: src/App.js [hunk 5]
```diff
@@ -1,26 +1,48 @@
...
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="body">
+        <div className="Buttons">
+          <button
+            className={`botton ${isActive === 'list' && 'bold-button'}`}
+            onClick={() => setIsActive('list')}
```
Comment (alejandronanez) [inline src/App.js:14]: You don't really need to use `useState` to set the active navigation item. React Router V4 already has this functionality baked in.

Take a look at [**NavLink**](https://reactrouter.com/web/api/NavLink)

**Review by alejandronanez — APPROVED** [review]
Nice work here @Yesika-Castillo and @andrewespejo!

I tested the functionality and works as expected 



Feel free to use `NavLink` and not `useState + Link + classes` to simplify things a bit.

## About styling
Let's not spend any more time writing any "complex" CSS, as weeks 7 and 8 are all about styling the application and this code will go away completely.

File: src/App.css [hunk 6]
```diff
@@ -1,22 +1,40 @@
...
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.no-decoration {
```
Comment (DNeosphere) [inline src/App.css:24]: Did you discuss a naming convention in order to maintain consistency in the  CSS classes? 
Take a look to this [BEM](http://getbem.com/l)

File: src/App.css [hunk 7]
```diff
@@ -1,22 +1,40 @@
...
+
+.body {
+  bottom: 0;
+  position: fixed;
+  width: 100%;
 }
 
-.App-link {
-  color: #09d3ac;
+h2 {
```
Comment (DNeosphere) [inline src/App.css:38]: try to style classes instead of general elements :). Take a look to the specificity concept of CSS

**Pull request merged**